resources/dicts/relationship_events/normal_interactions/neutral.json

```diff
@@ -123,11 +115,9 @@
 			"r_c seems to be oblivious to m_c glaring at {PRONOUN/r_c/object} from across camp.",
 			"m_c is sharpening {PRONOUN/m_c/poss} claws near r_c.",
 			"r_c notices that m_c knows a lot about the Dark Forest.",
-			"m_c says a fun fact about the Dark Forest to r_c.",
 			"r_c wonders when m_c learned so much about cat anatomy.",
 			"r_c has to snap m_c out of staring darkly at another Clanmate.",
 			"r_c sees m_c using a small animal bone to pick food from {PRONOUN/m_c/poss} teeth.",
-			"As r_c passes by, {PRONOUN/r_c/subject} {VERB/r_c/smell/smells} blood on m_c's fur.",

```

Comment:
Removed

Reply:
This one is a bit too similar to one I wrote in comfort decrease, so I removed it